service/integration/open_pbs/geopm_power_limit.py

```diff
@@ -0,0 +1,145 @@
+import sys
+
+PYTHON_PATHS = [
+        "/usr/lib/python3.6/site-packages",
+        "/usr/lib64/python3.6/site-packages"]
+
+for p in PYTHON_PATHS:
+    if p not in sys.path:
+        sys.path.insert(0, p)
+
+import os
+import json
+
+import pbs
+
+from geopmdpy import pio
+
+
+_SAVED_CONTROLS_FILE = "run/geopm-service/SAVE_FILES/saved_controls.json"
```

Comment:
Haven't decided on location and file name. We could put the file in the same location as the other save files. If so, perhaps the file could be named something like: "power-limit-save-control.json".

Alternatively, if we don't want to mix it with the other files, we could put it in a separate folder, for example: "/run/geopm-pbs-hooks/SAVE_FILES" or "/run/geopm-extensions/SAVE_FILES"

My preference would be to put it in a separate folder since the files are managed by separate modules/mechanisms.